runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/TemplatesSection/Categories/Categories.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FC, useMemo } from 'react';
+
+import useTranslations from '#src-app/hooks/useTranslations';
+import Typography from '#src-landing/components/Typography';
+
+import { categoriesTranslationKeys } from '../TemplatesSection.utils';
+import styles from './Categories.module.scss';
+import { CategoriesProps } from './Categories.types';
+
+const Categories: FC<CategoriesProps> = ({ 
+    selectedCategoryKey, 
+    handleCategoryChange,
+}) => {
+    const { translate } = useTranslations();
+
+    const categoriesList = useMemo(() => categoriesTranslationKeys.map((categoryKey) => {
+        // @ts-ignore
+        const categoryName = translate(`Landing.Templates.CategoriesBar.Category.${categoryKey}`);
```

Comment:
if u are absolutely sure just do assertion here istead of ts ignore. But personaly i would like to see ts guard checking if this translation key is `keyof typeof temaplatesTranslations`, if not provide fallback value

Reply:
done